clients/privacy-center/pages/consent.tsx

```diff
@@ -344,19 +344,25 @@ const Consent: NextPage = () => {
               fontWeight="semibold"
               textAlign="center"
             >
-              Manage your consent
+              {config.consent?.page.title}
             </Heading>
-            <Text
-              fontSize={["small", "medium"]}
-              fontWeight="medium"
-              maxWidth="624px"
-              textAlign="center"
-              color="gray.600"
-            >
-              When you use our services, you&apos;re trusting us with your
-              information. We understand this is a big responsibility and work
-              hard to protect your information and put you in control.
-            </Text>
+
+            {config.consent?.page.description_subtext?.map(
+              (paragraph, index) => (
+                <Text
+                  fontSize={["small", "medium"]}
+                  fontWeight="medium"
+                  maxWidth={624}
+                  textAlign="center"
+                  color="gray.600"
+                  data-testid={`description-${index}`}
+                  // eslint-disable-next-line react/no-array-index-key
+                  key={`description-${index}`}
```

Comment:
react keys are generally pretty important to get right, as they result in mysterious looking errors when they aren't right. I think we are okay for now here, since there really isn't a good key for this kind of thing. errors that could come up would be things like the element still lingering in the DOM when it shouldn't be, or you make a change to the element and suddenly there are multiples of that element. since this isn't a field that dynamically changes (except for maybe if you're hot reloading config.json? not sure) we are probably okay, but just want to flag what kind of risk we might bump into

Reply:
100% agree. In this case I'm under the impression our hands are tied because beyond the list index there's no other locally unique value to use for the key?